docs/content/standards/deepbookv3.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: DeepBookV3
+description: A central limit order book that offers features and functionality for marketplaces on Sui. Token exchanges leveraging the layer can feature transparency, a full range of trading options, and customer privacy.
+---
+
+DeepBook is a next-generation decentralized central limit order book (CLOB) built on Sui. DeepBook leverages Sui's parallel execution and low transaction fees to bring a highly performant, low-latency exchange on chain. 
+
+The latest version delivers new features including flash loans, governance, improved account abstraction, and enhancements to the existing matching engine. This version also introduces its own tokenomics with the [DEEP token](https://suiscan.xyz/mainnet/coin/0xdeeb7a4662eec9f2f3def03fb937a663dddaa2e215b8078a284d026b7946c270::deep::DEEP/txs), which you can stake for additional benefits. 
+
+DeepBook does not include an end-user interface for token trading. Rather, it offers built-in trading functionality that can support token trades from decentralized exchanges, wallets, or other apps. The available SDK abstracts away a lot of the complexities of interacting with the chain and building programmable transaction blocks, lowering the barrier of entry for active market making.
+
+## DeepBook tokenomics
+
+As mentioned, the DEEP token facilitates the operations of DeepBook. Initially, the tokens are required to pay for fees on the service. Future iterations will allow other forms of payment, like SUI, but owning, using, and staking DEEP will continue to provide benefits to active DeepBook traders on the Sui network.  
+
+Staked takers (participants that have DEEP tokens staked on Sui and place orders matched on the CLOB), for instance, can trade with fees as low as 0.25 basis points (bps) on stable pairs and 2.5 bps on volatile pairs. Staked makers (participants that have DEEP tokens staked on Sui and place limit orders on the CLOB) can earn rebates based on their staking share.
```

Comment:
Can we shorten the sentences? Something like "Users that stake DEEP can enjoy taker and maker incentives. Taker incentives can reduce trading fees by half, dropping them to as low as 0.25 basis points (bps) on stable pairs and 2.5 bps on volatile pairs. Maker incentives are rebates earned based on maker volume generated."